mlir/utils/performance/perfRunner.py

```diff
@@ -559,7 +558,7 @@ def generateMlirDriverCommandLine(self, rocmlir_gen_flags):
                            f"-transA={self.transA}",
                            f"-transB={self.transB}",
                            '--kernel-repeats', str(self.MLIR_N_REPEATS),
-                           f"--perf_config={self.perfConfig}"])
+                           f"--perf_config={self.perfConfig} "])
```

Comment:
Could you remove space changes? (Unless it's important that the space stays there)